docs/source/developer/contributing.rst

```diff
@@ -60,14 +60,21 @@ Submitting a Pull Request Contribution
 --------------------------------------
 
 Generally, an issue should be opened describing a piece of proposed work
-and the issues it solves before a pull request is opened.
+and the issues it solves before a pull request is opened. Your issue should 
+go through a brief triage process to ensure that it meets our definition 
```

Comment:
The way this is phrased doesn't make it clear that the person opening the issue probably won't be the one who does triage.